pages/index.js

```diff
@@ -1,7 +1,20 @@
+import EntryList from "@/components/EntryList";
+import useSWR from "swr";
+
 export default function HomePage() {
+  const { data:entries, isLoading } = useSWR("/api/entries", { fallbackData: [] });
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
```

Comment:
```suggestion
    return <p>Loading...</p>;
```

A better approach is to treat `<h1>` as the “main heading” of the page or section and use `<p>` or `<span>` for temporary/error messages.